### PR TITLE
Replaces the remaining(or most) of the right click verbs on machines with Altclick/Ctrlclick.

### LINKED ISSUE #17409: Revenants can interact with any machines that have right-click verbs
Cross-reporting from Paradise: https://github.com/ParadiseSS13/Paradise/issues/3357

We have fewer right-click verbs than they do, but ones that still exist:
- The gibber
- Chem grinders
- Food/slime processors

I probably missed a lot.

### PR DESCRIPTION
**Pull request by Firecage**
This PR does as it says in the title. I am pretty sure I got all machines, but if I missed one or two, please tell me.

PDA Painter: Altclick to eject PDA
Clone Pod: Altclick to Eject.
Doppler Array: Altclick to rotate
IV Drip: Altclick to eject bloodpack. Ctrlclick to toggle mode.
That map thingy. Ctrlclick to use it.
Gibber: Altclick to eject.
Juicer: Altclick to eject beaker. (yes, I had to do usr instead of user to get this shit to work without runtiming)
Processor: Altclick to eject.
PA: Altclick to rotate clockwise. Ctrlclick for counter clockwise.

Fixes #17409

### CODE REVIEW

**Comment by lzimann** [comment]
Replacing CtrlClick is probably a bad idea since it overrides the pull...

**Comment by ChangelingRain** [comment]
>ctrlclick
that's for DRAG! that's very unwise! you need to be able to drag your IV!

**Comment by Firecage** [comment]
Ah fuck, I forgot the IV was a draggable machine.

**Comment by jughu** [comment]
Make is ctrl middle mouse click

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
Don't remove the verb, new players aren't going to know these keys and I still think the verbs should remain

**Comment by XDTM** [comment]
Or add the Alt-click functions to the examine text.

"Alt-Click this to eject the clone early."

**Comment by MrStonedOne** [comment]
Nope, not good enough, the object tab must remain, it is my only saver when my keyboard is holding a cig, or my drink, and i need to perform one of these actions.

Right click requires one hand, ctrl/alt/shift/etc+click requires two hands.

and fucking really, what is the worst that could happen from leaving them in.

**Comment by bgobandit** [comment]
don't forget to check for ghosts

File: code/game/machinery/overview.dm [hunk 1]
```diff
@@ -1,16 +1,14 @@
 //#define AMAP
 
-/obj/machinery/computer/security/verb/station_map()
```
Comment (francinum) [inline code/game/machinery/overview.dm]: Does this even do anything anymore?

**Comment by PKPenguin321** [comment]
Definitely needs to either keep the verbs or add the alt click prompt in the examine.

**Comment by Firecage** [comment]
Eh, I guess @MrStonedOne makes a good point.

**Pull request closed without merging**